### DISCUSSION TITLE
Perfect-Vision Global illumination providing too perfect vision

### DISCUSSION THREAD

**Discussion by Tireddadgamer** [dev7355608/perfect-vision#300] [General]
When using global illumination, tokens can see through walls both landscape and regular. They are still stopped by the walls, but they do not block line of sight. When "provides vision" is set to off for global illumination or in night mode and the players have to use other light sources it works perfectly. Am I missing a check box somewhere?

**Comment by dev7355608** [comment]
*Provides Vision* enables the token to see the illuminated area regardless of line-of-sight restrictions. You likely don't want to enable that. Are you saying with *Provides Vision* off global illumination isn't working and tokens need to use other light sources to be able to see?

**Comment by Tireddadgamer** [comment]
ah man...I swear I tried having "provides vision" unclicked and had no
vision before. now it's working perfectly. Dagnabbit. I'm sorry for wasting
your time.
